plugins/inputs/system/README.md

```diff
@@ -8,8 +8,12 @@ and number of users logged in. It is similar to the unix `uptime` command.
 ```toml
 # Read metrics about system load & uptime
 [[inputs.system]]
-  # no configuration
+  ## Uncomment the following line to disable uptime collection
+  # skip_uptime = true
```

Comment:
Let's make this a filter. I think just an exclude filter is sufficient. Allowed values would be `uptime` and `users`.
```
filter.NewIncludeExcludeFilter([]string{}, SomethingExclude)
```
The docker input plugin uses filters and may be a good reference.